src/modules/rewards/components/RewardsTabsContainer.tsx

```diff
@@ -33,7 +33,7 @@ const RewardsTabsContainer: FC<RewardsTabsContainerProps> = ({ activeTab, handle
         />
 
         {activeTab === 'dashboard' && <DashboardSection onGetStarted={() => handleSetActiveTab('activity')} />}
-        {activeTab === 'activity' && <RewardsActivitiesSection />}
+        {activeTab === 'activity' && <DailyRewardsSection />}
```

Comment:
Here we should not render only the daily rewards but all the rewards related stuff. Current approach is not correct. There is no need to use separate component outside tabs.